get_fs_util.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Pulls in fs_util tool for use in remote exec ingestion, checkout, and CAS ops
+
+pants_release="1.4.0"
```

Comment:
Technically the pants release here is 1.4.0dev23, and fs_util is just a part of the pants release.